content/developer/writing_plugins/creating-your-own-plugin.md

```diff
@@ -615,7 +615,7 @@ Several attributes under `ctx.plugin` can be used to access details about the pl
 
 ## Testing Your Plugin
 
-In most cases, the recommendation is to test your plugin's logic using local workflows, and only then run them as part of a Cloudify deployment. We have supplied you with a nice and tidy
+In most cases, the recommendation is to test your plugin's logic using local workflows, and only then run them as part of a {{< param product_name >}} deployment. We have supplied you with a nice and tidy
 decorator to do just that. The `cloudify-common`'s `test_utils` package enables you to do that. It is intuitive to use, and an example is provided below:
```

Comment:
`cloudify-common` - not sure if we can change that.

Reply:
no, we cannot.